src/cheri_cap_common.sail

```diff
@@ -488,6 +488,60 @@ function setCapBounds(cap, base, length) : (Capability, CapAddrBits, CapAddrBits
   (exact, newCap)
 }
 
+/* XXX this should go in prelude.sail but putting here for now to avoid submodule faff */
```

Comment:
I don't suppose you feel like doing the necessary faffing?

Reply:
Not especially... I should probably go raise a PR for sail to add it next to https://github.com/rems-project/sail/blob/1cc693a30db7283d2f6d96360b1bcde94d1f54fb/lib/vector.sail#L92 and then everyone can stop adding their own versions.